src/test/java/no/ssb/dlp/pseudo/service/sid/ExternalSidServiceTest.java

```diff
@@ -46,6 +46,32 @@ public void testInvokeMultiFnr() {
 
         verify(sidClient, times(1)).lookup(any(MultiSidRequest.class));
     }
+
+    @Test
+    public void testInvokeSingleSnr() {
+        when(sidClient.lookup(any(SidRequest.class))).thenReturn(Publishers.just(
+                SidInfo.builder().snr("0001ha3").fnr("11854898347").build())
+        );
+        sidService.lookupSnr("0001ha3", Optional.ofNullable(null));
+
+        verify(sidClient, times(1)).lookup(any(SidRequest.class));
+    }
+
+    @Test
+    public void testInvokeMultiSnr() {
+        // sidService should call our sidClient mock
+        when(sidClient.lookup(any(MultiSidRequest.class))).thenReturn(Publishers.just(
+                MultiSidResponse.builder().mapping(MultiSidResponse.Mapping.builder()
+                        .snr(Arrays.asList("0001ha3"))
+                        .snr(Arrays.asList("0006kh2"))
+                        .fnr(Arrays.asList("11854898347"))
```

Comment:
Instead of `Arrays.asList` prefer using `List.of`as it returns a copy of the reference passed to it. This could also be written a bit more concisely:
```diff
- .snr(Arrays.asList("0001ha3"))
- .snr(Arrays.asList("0006kh2"))
- .fnr(Arrays.asList("11854898347"))
+ .snr(List.of("0001ha3", "0006kh2"))
+ .fnr(List.of("11854898347"))
```